### DISCUSSION TITLE
Is it possible to pop up an exisiting note when creating a new note from a template?

### DISCUSSION THREAD

**Discussion by francisHugo** [SilentVoid13/Templater#1439] [Help] [answered]
**What I am trying to do is:**

template is ready, new notes could be created from the template.  

Now, I would like to have `an instruction note` pops up to explain how to use the template when the template is applying.

**Reason for this:**

I had some templates which i thought are quite useful (thanks for the template plugin, and many thanks to the plugin creator), I shared these template with frineds who have no any experiences of using Obsidian, so, I planned to share some `instruction note` with templates to them, so, when they created new notes from a template, `an instruction note` will pop up to explain how to use the template.

TIA

**Comment by Zachatoo** [comment, marked as answer]
Here's one way to do that.

```js
<%*
const note1 = tp.file.find_tfile("filename1");
app.workspace.getLeaf(true).openFile(note1);
-%>
```

**Reply by francisHugo** [reply]
Thanks a lot. 

This is the magic I am looking for.  Many Thanks!
